crates/fuel-core/src/schema/block.rs

```diff
@@ -165,7 +165,7 @@ impl Header {
 
     /// Merkle root of message receipts in this block.
     async fn message_receipt_root(&self) -> Bytes32 {
```

Comment:
We should also rename this function, as it's what's used to generate the fields of the gql api.